service/src/main/java/greencity/mapping/UserToUserProfileUpdateDto.java

```diff
@@ -0,0 +1,31 @@
+package greencity.mapping;
+
+import greencity.dto.AddressDto;
+import greencity.dto.UserProfileUpdateDto;
+import greencity.entity.user.User;
+import greencity.entity.user.ubs.Address;
+import org.modelmapper.AbstractConverter;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class UserToUserProfileUpdateDto extends AbstractConverter<User, UserProfileUpdateDto> {
+    @Override
+    protected UserProfileUpdateDto convert(User user) {
+        List<Address> addresses = user.getAddresses();
+        List<AddressDto> addressDtoList = addresses.stream()
```

Comment:
Is it need to create list of Address? Why not write user.getAddresses .stream()... ?